crates/containerd-shim-wasm/src/sandbox/sys/windows/stdio.rs

```diff
@@ -0,0 +1,42 @@
+use std::io::{Error, ErrorKind, Result};
+use std::os::windows::prelude::{AsRawHandle, IntoRawHandle, OwnedHandle};
+
+type RawFd = libc::c_int;
+
+pub static STDIN_FILENO: RawFd = 0;
+pub static STDOUT_FILENO: RawFd = 1;
+pub static STDERR_FILENO: RawFd = 2;
```

Comment:
for reference https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/dup-dup2?view=msvc-170

Reply:
yes, that's what I took from it 👍  I was verifying and left a note here for future reference